src/components/WwwFrame/LendMenu/LendMegaMenu.vue

```diff
@@ -3,11 +3,33 @@
 		<router-link
 			v-if="showMGUpsellLink"
 			to="/monthlygood"
-			class="tw-inline-flex tw-gap-0.5 tw-py-2 tw-mb-2 tw-font-medium"
 			@click.native="trackMgLinkClick"
+			:class="{ 'hover:tw-no-underline': newMgEntrypoint}"
 		>
-			Lend monthly
-			<kv-material-icon :icon="mdiArrowRight" class="tw-w-3 tw-h-3" />
+			<div class="tw-mb-4 tw-inline-flex tw-items-center" v-if="newMgEntrypoint">
```

Comment:
It looks like these two new blocks are identical, might be worth creating a shared component in a new file in the `LendMenu` folder.

Reply:
+1

Reply:
Not identical, but I like this way. Thanks guys!